.github/workflows/CI.yml

```diff
@@ -94,7 +94,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        chpl-version: ['1.26.0', '1.27.0']
+        chpl-version: ['1.26.0', '1.27.0', '1.28.0']
```

Comment:
I am not sure we need 1.26 still since we are only supporting one previous release of chapel. With that said, I don't think having it here is an issue at all.

Reply:
I'm personally not sure what the policy is for which versions to support. If somebody wants to create an issue to remove 1.26 support, I'll be happy to remove it in a followup PR.